app/src/styles.sass

```diff
@@ -142,18 +138,25 @@ $itero-mat-theme: mat-light-theme(( color: ( primary: $itero-mat-primary, accent
 @include angular-material-typography($itero-mat-typography)
 
 
-.sidemenu
+
+.infomenu
   border-left:1px solid #704f6a
   border-top: 1px solid #704f6a
   border-bottom: 1px solid #704f6a
   background-color: #d5b6d0
   float: right
-  height: 100%
-  margin-right: -16px
   margin-left: 3px
-  box-shadow: -2px 0px 2px 0 rgba(255, 204, 244, 0.9)
-  .helpbutton
-    text-align: center
+  box-shadow: -5px 0px 5px 0 rgba(255, 204, 244, 0.7)
+  width: 80%
+  margin-right: 0
+  position: absolute
+  right: 0
+  display: flex
+  flex-flow: row nowrap
+  .infomenuicon
+    display: flex 
+    .helpbutton
+      align-self: flex-start
```

Comment:
These 4 lines are not used because there is a typo in the class name in the HTML file (infomenui**n**con). This means that you can remove them.